ports/samd/boards/ADAFRUIT_FEATHER_M0_EXPRESS/pins.csv

```diff
@@ -8,9 +8,9 @@ PIN_PB03,LED_RX
 
 PIN_PA11,D0
 PIN_PA10,D1
-PIN_PA14,D2
-PIN_PA09,D3
-PIN_PA08,D4
+PIN_PA08,FLASH_MOSI
+PIN_PA14,FLASH_MISO
+PIN_PA09,FLASH_SCK
```

Comment:
Should these be renamed?  There's no longer any need to have them accessible in `_boot.py` as these names.

It's a little inconsistent that these are called FLASH and the others below are QSPI.

Reply:
They are used in samd_spiflash.c. And they are called FLASH_xxxx, to make clear that they are connected to the on-board flash chip, and user accessible SPI pins of a board.